.github/workflows/main.yml

```diff
@@ -155,3 +155,11 @@ jobs:
           CHROMIUM: true
         run: |
           npm run unit
+
+      - name: Run unit tests on Firefox
+        env:
+          FIREFOX: true
+        run: |
+          $env:PUPPETEER_PRODUCT='firefox'
+          $env:MOZ_WEBRENDER='0'
```

Comment:
These can be added to to `env` above. (We only use `$env:` for environment variables which we only need for one (but not all) command within the `run` block)

Reply:
Oh, that is actually a left over from testing. I corrected that.